### ISSUE TITLE
steps_per_epoch = 3

### ISSUE THREAD

**Issue by elisim** [NitishaS-812k/Predicting-ages-of-bones-from-X-Rays#1]
Hey,

Nice code! thanks.
Why you use steps_per_epoch = 3 in your fit?
This is not a typical training setting:

https://datascience.stackexchange.com/questions/47405/what-to-set-in-steps-per-epoch-in-keras-fit-generator

**Comment by NitishaS-812k** [comment]
Hey,
Thanks for pointing this out! This is my first time working with ImageDataGenerators. I mistakenly assumed that this number is set arbitrarily. However, setting it to 315, improved accuracy greatly!
